app/views/categories/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<header>
+  <nav class="nav navbar-light bg-primary">
+    <div class="container py-1 px-3 d-flex justify-content-between align-items-center">
+    <div></div>
+    <h1 class="fs-6 fw-normal text-white pt-2 pg-title">CATEGORIES</h1>
+    <small><%= link_to 'Log Out', destroy_user_session_path, method: :delete, class: "text-white text-decoration-none bg-transparent border-0 fw-normal" %></small>
```

Comment:
- [ ] When I click on the logout, I got the following error, consider fixing it and make sure works properly
>![image](https://user-images.githubusercontent.com/53402600/227622882-6092af9d-378e-415a-9b97-78b1e02511a9.png)